docs/book/v3/file.md

```diff
@@ -203,13 +197,11 @@ The following set of options are supported:
 > It is generally a better idea to supply an internal filename to avoid
 > security risks.
 
-`RenameUpload` does not support an array of options like the`Rename` filter.
-When filtering HTML5 file uploads with the `multiple` attribute set, all files
-will be filtered with the same option settings.
+`RenameUpload` can only operate on one file at a time.
 
 ### Usage Examples
 
-Move all filtered files to a different directory:
+Move all filtered files to a different directory in a Laminas MVC application:
```

Comment:
I do not understand the addition. Does not this also work in simple scripts without laminas-mvc?

Reply:
I've removed that addition and changed one of the examples to use `$_FILES` directly.

Also, dropped the second, nullable arg to the constructor in the examples which is unnecessary and probably will just cause confusion for users